prisma/schema.prisma

```diff
@@ -23,6 +23,74 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+// Achhievement
+// A final achievement can consist of several under achievements
+model achievement_template {
+  id                            Int                       @id @default(autoincrement())
+  metrics                       String[]                  @default([])
+  templateFor                   template_for_enum
+  // Groups defines the "final-achievement"
+  group                         String                    @unique @db.VarChar
+  // Group order tells in which place a subordinate step / achievement is
+  groupOrder                    Int                      
+  // the final achievement is marked with isLast 
+  isLast                        Boolean
+  // what type of achievement is it
+  type                          achievement_type_enum
+  name                          String                    @db.VarChar
+  subtitle                      String                    @db.VarChar
+  description                   String                    @db.VarChar
+  image                         String                    @db.VarChar
+  // some achievements show actions that a user can/must perform next 
+  actionDescription             String?                   @db.VarChar
+  actionRedirectLink            String?                   @db.VarChar
+  actionType                    action_type_enum?
+  // defines what value must be reached to receive the achievement 
+  condition                     String                    @db.VarChar
```

Comment:
what kind of string is this? I guess it must be in some specific format? (maybe worth mentioning in a comment)